src/cgame/cg_players.c

```diff
@@ -1912,6 +1912,8 @@ static void CG_BreathPuffs(centity_t *cent, refEntity_t *head)
 		return;
 	}
 
+	ci = &cgs.clientinfo[cent->currentState.number];
```

Comment:
Because it happens for `cent->currentState.number >= 64` which is `ET_CORPSE` which early outs here:
https://github.com/etlegacy/etlegacy/blob/8f97f0332b18e29c02967f61b508a8eb38974ca1/src/cgame/cg_players.c#L1898-L1901